StaticSVDBasisGenerator.C

```diff
@@ -30,7 +31,8 @@ StaticSVDBasisGenerator::StaticSVDBasisGenerator(
                                            samples_per_time_interval,
                                            max_basis_dimension,
                                            sigma_tolerance,
-                                           debug_algorithm));
+                                           debug_algorithm,
+                                           output_rightSV));
```

Comment:
@kevinhkhuynh, Introducing output_rightSV is good, but it does not complete the task of enabling temporal basis write for "static" SVD, yet. Is it still in progress?

Reply:
Hi Youngsoo. If the user sets output_rightSV to true, temporal basis is outputted. Otherwise, output_rightSV defaults to false and nothing is outputted. Temporal basis was already there, but wasn't being outputted before because in StaticSVDSampler.h, updateRightSV defaults to false, and the user never had a way to set it to true, but now they can with output_rightSV.